flink-sequence-features/README.md

```diff
@@ -0,0 +1,85 @@
+# Overview
+
+This example shows how to generate sequence feature using FeatHub's
+`COLLECT_LIST` aggregation function. It involves the following steps:
+
+1. Read a stream of real-time browse events from a Kafka topic.
+
+   Each browse event has the following fields:
+   - user_id, unique identifier of a user.
+   - item_id, unique identifier of the item that is browsed by the user.
+   - event_type, the type of the event, e.g., click, exposure.
+   - timestamp, time when this purchase is made.
+
+2. Compute a sequential feature representing the click history of a user. This
```

Comment:
Would it be simpler to say the following

For each event received from Kafka, computes and appends an extra feature which shows a list of last 10 item that the given user has clicked. This aggregation computation is achieved using `OverWindowTransform`.